Library/OcMainLib/OpenCoreMisc.c

```diff
@@ -259,37 +258,26 @@ OcToolDescribeEntry (
 {
   EFI_STATUS          Status;
   CHAR16              DescPath[OC_STORAGE_SAFE_PATH_MAX];
-  OC_STORAGE_CONTEXT  *Storage;
   BOOLEAN             HasIcon;
   BOOLEAN             HasLabel;
 
-  Storage  = (OC_STORAGE_CONTEXT *) Context;
   HasIcon  = FALSE;
   HasLabel = FALSE;
 
   if (LabelData != NULL && LabelDataSize != NULL) {
     *LabelData     = NULL;
     *LabelDataSize = 0;
 
-    if (ChosenEntry->Type == OC_BOOT_RESET_NVRAM) {
-      Status = OcUnicodeSafeSPrint (
-        DescPath,
-        sizeof (DescPath),
-        OPEN_CORE_LABEL_PATH "ResetNVRAM.%a",
-        LabelScale == 2 ? "l2x" : "lbl"
-        );
-    } else {
-      Status = OcUnicodeSafeSPrint (
-        DescPath,
-        sizeof (DescPath),
-        OPEN_CORE_TOOL_PATH "%s.%a",
-        ChosenEntry->PathName,
-        LabelScale == 2 ? "l2x" : "lbl"
-        );
-    }
+    Status = OcUnicodeSafeSPrint (
+      DescPath,
+      sizeof (DescPath),
+      OPEN_CORE_TOOL_PATH "%s.%a",
```

Comment:
What will this have in case of Reset NVRAM or CSR code? I get a feeling it may contain human readable text, which sounds wrong.

Reply:
This is no longer called for internal tools, at least as I have changed it so far. That is what I mean about that other line, getting their label from the pre-loaded internal ones even if generic labels are not set, replacing this code.